yt/visualization/profile_plotter.py

```diff
@@ -115,7 +115,32 @@ def data_object_or_all_data(data_source):
     return data_source
 
 
-class ProfilePlot(PlotContainer):
+class PlotContainerWithFieldLabels(PlotContainer):
+    # A common ancestor to ProfilePlot and PhasePlot
+    # that defines field labels
+
+    def _get_field_label(self, field, field_info, field_unit, fractional=False):
+        field_unit = field_unit.latex_representation()
+        field_name = field_info.display_name
+        if isinstance(field, tuple):
+            field = field[1]
+        if field_name is None:
+            field_name = field_info.get_latex_display_name()
+        elif field_name.find("$") == -1:
+            field_name = field_name.replace(" ", r"\ ")
+            field_name = r"$\rm{" + field_name + r"}$"
+        if fractional:
+            label = field_name + r"$\rm{\ Probability\ Density}$"
+        elif field_unit is None or field_unit == "":
+            label = field_name
+        elif r"\frac" in field_unit:
+            label = field_name + r"$\ \ \left(" + field_unit + r"\right)$"
```

Comment:
Why this logic bloc here? I thought the result of `(x = 1)` and `\left( x = 1 \right)` were the same?

Reply:
you're right. I just cut-pasted the function without even reading it since it had two identical copies but indeed it could be simplified.

Reply:
I removed this from the PR because the real fix I was going for is actually a single line diff